connect_cmd.go

```diff
@@ -52,8 +52,10 @@ func (connectResult ConnectResult) Error() string {
 	return result
 }
 
-// beforeConnectAction ensures that user has supplied a correct CLI options
-// and there is no conflict between provided options
+// beforeConnectAction ensures that user has supplied correct CLI options
+// and there is no conflict between provided options. When there is anything
+// wrong, then this function has to return cli.Exit() error. The exit codes
+// are defined in constants.go module
```

Comment:
I just have some word-smithing suggestions:

Please consider the following updates:

// beforeConnectAction ensures that **the** user has supplied **the** correct CLI options
// and there is no conflict between ~~provided options~~ **them**. When there is anything
// wrong this function ~~has to return~~ **will invoke** cli.Exit() **with an appropriate** error **message and
// error code**.
// The exit codes are defined in **the** constants.go module

Reply:
Fixed according your suggestions. Thanks.